src/js/common/schemaform/helpers.js

```diff
@@ -245,6 +245,10 @@ export function stringifyFormReplacer(key, value) {
     if (fields.length === 0 || fields.every(field => value[field] === undefined)) {
       return undefined;
     }
+    // autosuggest widgets save value and label info, but we should just return the value
+    if (value.widget === 'autosuggest') {
```

Comment:
We need to have both the id and the label in the form data, in case a user comes back to the autosuggest field. But we don't want to send both. So I'm adding some extra data to tag this as an autosuggest field and convert it to the id.